lib/rtc_peer.js

```
@@ -267,7 +269,29 @@ export class RTCPeer extends EventEmitter {
         });
     }
     onTrack(ev) {
-        this.emit('stream', new MediaStream([ev.track]), this.mediaMap[ev.transceiver.mid]);
+        const mid = ev.transceiver.mid;
+        const trackInfo = this.mediaMap[mid];
+        if (trackInfo) {
+            this.emit('stream', new MediaStream([ev.track]), trackInfo);
+        }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)